src/components/templates/TemplateSearchPortal.vue

```diff
@@ -1,5 +1,45 @@
 <template>
     <div class="custom-wrap">
+        <div class="so-btn">
+            <AtomButton @click.native="showSummary">
+                {{ summary.btn }} Summary
+            </AtomButton>
+        </div>
+        <br />
+        <div class="so-summary" v-show="summary.show">
+            <p class="so-total">Total Request : {{ summary.totalRequest }}</p>
+            <hr />
+            <div class="so-priority">
+                <p>High : {{ summary.high }}</p>
+                <p>Low : {{ summary.low }}</p>
+                <p>Medium : {{ summary.medium }}</p>
+            </div>
+            <hr />
+            <div class="so-status">
+                <p>
+                    <span>Not Set :</span> <span>{{ summary.status[0] }}</span>
```

Comment:
We can remove this as these number will always be zero

Reply:
done